src/main/java/hudson/plugins/collabnet/orchestrate/BuildNotifierDescriptor.java

```diff
@@ -155,4 +155,16 @@ public boolean canInheritAuth() {
     public static TeamForgeShare.TeamForgeShareDescriptor getTeamForgeShareDescriptor() {
         return TeamForgeShare.getTeamForgeShareDescriptor();
     }
+
+    /**
+     * Validates that the user provided a webhook url.
+     *
+     * @param webhookUrl
+     *            the webhookUrl provided by the user
+     * @return whether or not the validation succeeded
+     */
+    public FormValidation doCheckWebhookUrl(@QueryParameter String webhookUrl) {
+        return FormValidation.validateRequired(webhookUrl);
```

Comment:
This should not be required for eventq users.

Reply:
This will be validated only if the user choose 'Notify teamforge' option and for the Webhook Url field alone.

Reply:
I thought it was not required, i.e. could be generated from CTF url?